### PR TITLE
feat(ui): add support for advanced options (#716)

### LINKED ISSUE #716: Implement specific (multi repository) package upgrades / downgrades
**Is your feature request related to a problem? Please describe.**
As user I want to change the version of an installed package (and even install the package from a different repository)

**Describe the solution you'd like**
Introduce a new repository dropdown for not yet installed and already installed packages and add a version select to already installed packages, which changes the package version.
I also want to be able to change the auto upgrade checkbox for already installed packages.

**Describe alternatives you've considered**
It is already possible to install / configure specific versions from the cli (`glasskube install keptn --version=v2.0.0-rc.1+1`)

**Additional context**
I already created a mockup of the app detail page:

Glasskube package version and repo switch

### PR DESCRIPTION
**Pull request by christophenne**
Closes #716

## ✅ Checks
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required

### CODE REVIEW

**Review by pmig — CHANGES_REQUESTED** [review]
I always like advanced options :nerd_face:

I encountered two problems:

- The Advanced options look off in dark mode:

image

- I was a able to create following bug:

1. Have multiple repositories, but no default repository
2. install a package that has no `repositoryName` set
3. Enable advanced option
4. Go to package detail and change version
5. Package detail package will break with following error:

> `An error occurred fetching repository : resource name may not be empty`

File: internal/web/templates/pages/settings.html [hunk 1]
```diff
@@ -28,6 +28,38 @@ <h2 class="text-reset">Package Repositories</h2>
...
+              class="form-check-input"
+              type="checkbox"
+              role="switch"
+              name="enableAdvancedOptions"
+              id="enableAdvancedOptions" />
+            <label class="form-check-label mx-1 fw-semibold" for="enableAdvancedOptions">Advanced Options</label>
+          </div>
+          <i class="bi bi-exclamation-triangle-fill text-warning me-1"></i>
+          It is not recommended to use these except for testing / maintaining a Glasskube package. These should only
+          ever be used in dev/test environments, and never in production. <b>Use at your own risk!</b>
```
Comment (kosmoz) [inline internal/web/templates/pages/settings.html:45]: I think there is missing a short description about what the advanced options do.

Also, `<strong>` should be preferred over `b`.

**Review by kosmoz — COMMENTED** [review]
Just a small comment in addition to what has already been said. 

Other than that, looking good.

**Review by pmig — APPROVED** [review]
The only small improvement I can think of is that if a package is installed from a specific repository / in a specific version that these values are the default values for the select inputs see:

image

File: internal/web/templates/components/pkg-config-advanced.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            id="advanced-pkg-install-repository"
+            name="repositoryName"
+            required
+            hx-select="#advanced-options"
+            hx-swap="outerHTML"
+            hx-target="#advanced-options"
+            hx-get="/packages/{{ .Manifest.Name }}/configure/advanced"
+            hx-include="#advanced-pkg-install-version">
+            {{ range .Repositories }}
+              <option value="{{ .Name }}" {{ if eq .Name $.RepositoryName }}selected{{ end }}>
```
Comment (kosmoz) [inline internal/web/templates/components/pkg-config-advanced.html:30]: I noticed that the current repository is not selected initially, even though it should be, according to this.

**Review by kosmoz — CHANGES_REQUESTED** [review]
I noticed a minor bug, but otherwise looking good.

**Pull request merged**